concert/experiments/xraytube.py

```diff
@@ -68,33 +66,35 @@ async def __ainit__(self, walker, flat_motor, radio_position, flat_position, cam
         :param num_projections: Number of projections.
         :type num_projections: int
         """
-        XrayTubeMixin.__init__(self, xray_tube)
-        await BaseRadiography.__ainit__(
-            self, walker=walker,
-            flat_motor=flat_motor,
-            radio_position=radio_position,
-            flat_position=flat_position,
-            camera=camera, num_flats=num_flats,
-            num_darks=num_darks,
-            num_projections=num_projections,
+        self._xray_tube = xray_tube
+        await super().__ainit__(
+            walker,
+            flat_motor,
+            radio_position,
+            flat_position,
+            camera,
+            num_flats,
+            num_darks,
+            num_projections,
             separate_scans=separate_scans
         )
 
 
-class SteppedTomography(XrayTubeMixin, BaseSteppedTomo):
+class ContinuousTomographyLogic(XrayTubeMixin, imaging.ContinuousTomographyLogic):
     """
-    Stepped tomography experiment.
+    Continuous tomography logic class which needs to be combined with one of the local or remote
+    mixins for DAQ.
     """
     async def __ainit__(self, walker, flat_motor, tomography_motor, radio_position, flat_position,
                         camera, xray_tube, num_flats=200, num_darks=200, num_projections=3000,
-                        angular_range=360 * q.deg, start_angle=0 * q.deg, separate_scans=True):
+                        angular_range=180 * q.deg, start_angle=0 * q.deg, separate_scans=True):
```

Comment:
The 360 was here for a reason.

Reply:
copy-paste error, good that we review ;-)

Reply:
The continuous and stepped classes are swapped, so it is actually like before.